package.json

```diff
@@ -2,6 +2,7 @@
   "name": "notifirehq",
   "private": true,
   "scripts": {
+    "nx:build": "pnpm nx run-many --target=build --all",
```

Comment:
I think we can change the `build` we currently have instead of adding a specific command for that. Another not really related task, is that maybe we can add `pnpm` as a devDependency to the `package.json` file so we don't need to prefix with `pnpm` inside the scripts section in package.json